bindings/python/htcondor2/_job_event_log.py

```diff
@@ -1 +1,79 @@
+import datetime
 
+from .htcondor2_impl import _handle as handle_t
+
+from ._job_event import JobEvent
+
+from .htcondor2_impl import (
+    _job_event_log_init,
+    _job_event_log_next,
+    _job_event_log_close,
+)
+
+class JobEventLog():
+
+    def __init__(self, filename : str):
+        self._deadline = 0
+        self._filename = filename
+        self._handle = handle_t()
+
+        if not isinstance(filename, str):
+            raise TypeError("filename must be a string")
+        if len(filename) == 0:
+            raise ValueError("filename must not be empty")
+        _job_event_log_init(self, self._handle, filename)
+
+
+    def events(self, stop_after : int = None) -> 'JobEventLog':
+        if stop_after is None:
+            self._deadline = 0
+        elif isinstance(stop_after, int):
+            now = datetime.datetime.now(datetime.timezone.utc).timestamp()
```

Comment:
time.time() appears to give the same value with easier-to-understand code.

Reply:
Yeah, not sure why I thought I needed a `datetime.datetime` object here.  Will fix.